crates/environ/src/vmoffsets.rs

```diff
@@ -591,6 +607,19 @@ impl VMOffsets {
     }
 }
 
+/// Offsets for `VMExternRefActivationsTable`.
+impl VMOffsets {
+    /// Return the offset for `VMExternRefActivationsTable::next`.
+    pub fn vm_extern_ref_activation_table_next(&self) -> u32 {
```

Comment:
I don't actually see this end up getting used anywhere, was this perhaps from a previous iteration?